src/lib.rs

```diff
@@ -426,8 +426,8 @@ fn json_arr_index(ctx: &Context, args: Vec<String>) -> RedisResult {
     let key = args.next_string()?;
     let path = backwards_compat_path(args.next_string()?);
     let json_scalar = args.next_string()?;
-    let start = args.next().map(|v| v.parse()).unwrap_or(Ok(0))?;
-    let end = args.next().map(|v| v.parse()).unwrap_or(Ok(usize::MAX))?;
+    let start = args.next().map(|v| v.parse::<i64>()).unwrap_or(Ok(0))?;
```

Comment:
Please remember to `cargo fmt` before PRs (and in general).

Reply:
done